Pcap++/header/PcapLiveDevice.h

```diff
@@ -222,60 +222,60 @@ namespace pcpp
 		/**
 		 * @return The type of the device (libPcap, WinPcap or a remote device)
 		 */
-		virtual LiveDeviceType getDeviceType() { return LibPcapDevice; }
+		virtual LiveDeviceType getDeviceType() const { return LibPcapDevice; }
 
 		/**
 		 * @return The name of the device (e.g eth0), taken from pcap_if_t->name
 		 */
-		inline const char* getName() { return m_Name; }
+		const char* getName() const { return m_Name; }
 
 		/**
 		 * @return A human-readable description of the device, taken from pcap_if_t->description. May be NULL in some interfaces
 		 */
-		inline const char* getDesc() { return m_Description; }
+		const char* getDesc() const { return m_Description; }
 
 		/**
 		 * @return True if this interface is a loopback interface, false otherwise
 		 */
-		inline bool getLoopback() { return m_IsLoopback; }
+		bool getLoopback() const { return m_IsLoopback; }
 
 		/**
 		 * @return The device's maximum transmission unit (MTU) in bytes
 		 */
-		virtual inline uint16_t getMtu() { return m_DeviceMtu; }
+		virtual uint16_t getMtu() const { return m_DeviceMtu; }
 
 		/**
 		 * @return The device's link layer type
 		 */
-		virtual inline LinkLayerType getLinkType() { return m_LinkType; }
+		virtual LinkLayerType getLinkType() const { return m_LinkType; }
 		/**
 		 * @return A vector containing all addresses defined for this interface, each in pcap_addr_t struct
 		 */
-		inline std::vector<pcap_addr_t>& getAddresses() { return m_Addresses; }
+		std::vector<pcap_addr_t>& getAddresses() { return m_Addresses; } // TODO: Should getAddress really return non-const reference?
```

Comment:
Pay attention on comment

Reply:
This list is read-only and shouldn't be changed by the caller so I guess the answer to your question is "no" - we should probably make it a const reference

Reply:
The changes have been made